src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx

```diff
@@ -22,35 +24,24 @@ export class ArtworkDetailsAboutTheWorkFromPartner extends React.Component<
     if (!additional_information) {
       return null
     }
-    const locationNames =
-      partner &&
-      partner.locations &&
-      partner.locations.length > 0 &&
-      filterLocations(partner.locations)
+
+    const locationNames = get(
+      partner,
+      p => filterLocations(p.locations),
+      []
+    ).join(", ")
```

Comment:
I personally find this to be less readable than the previous version. Does it functionally protect against more?

Reply:
I think  prettier breaking it out onto multiple lines makes it a bit weird, but the previous version was quite unwieldy, with two blocks required to ensure that what's returned is valid in order to join: 

```js
const locationNames =
      partner &&
      partner.locations &&
      partner.locations.length > 0 &&
      filterLocations(partner.locations)

...

{locationNames &&
   locationNames.length > 0 && (
     <Serif size="3" color="black60">
       {locationNames.join(", ")}
     </Serif>
)}
```

which now becomes basically a one-liner: 

```js
const locationNames = get(partner, p => filterLocations(p.locations), []).join(", ")
```

Reply:
I personally still prefer the former, but that doesn’t really matter for this one example. My main point is that I don’t feel like we should necessarily go ahead and replace all places with this, but make a good judgement on wether or not this runtime safety is really necessary.